lib/ace/ext/papeeria/snippets/EquationSnippets.coffee

```diff
@@ -0,0 +1,119 @@
+define((require, exports, module) ->
```

Comment:
CapitalizedNames in CS/JS files are not quite common, at least in Ace and Papeeria code. Pls rename back to `equation_snippets.coffee`